### DISCUSSION TITLE
Possible to rename a repository?

### DISCUSSION THREAD

**Discussion by JonathanEbner** [sebastian-hofstaetter/teaching#148] [Advanced IR: Exercise Q&A] [answered]
I accidentally created a repository called "air-2024-test". I can't rename it to group-33, and I can't create a new one either. Can we just use the one called "air-2024-test" or can you rename it for us?
Thanks in advance!

**Comment by peterknees** [comment, marked as answer]
I changed the name of the repository to air-2024-group-33. It's not reflected on Classroom (yet?) so your colleagues might still have to join "test" in order to be in "group-33" (please check which is available or if anything was broken by renaming).
